src/uu/ls/src/ls.rs

```diff
@@ -283,6 +291,23 @@ fn sort_entries(entries: &mut Vec<PathBuf>, options: &getopts::Matches) {
     }
 }
 
+#[cfg(windows)]
+fn should_display(entry: &DirEntry, options: &getopts::Matches) -> bool {
+    let ffi_name = entry.file_name();
+    let name = ffi_name.to_string_lossy();
+    let hidden_by_ntfs = is_hidden(&entry.path()).unwrap();
```

Comment:
to avoid duplication, this should be set to false to all platforms
and make this check only on Windows

Or you could create new functions to avoid code duplication

Reply:
Yeah this is what I just suggested in a command I will try

Reply:
Code duplication issues should be fixed